docs/custom-types.md

```diff
@@ -75,14 +73,15 @@ custom_id: <custom ID>
 
 *Example:*
 
-To set up a consumer, first we need a plugin and a credential:
+This resource allows configuring Consumers in Kong
 
 ```yaml
 apiVersion: configuration.konghq.com/v1
 kind: KongConsumer
 metadata:
   name: consumer-team-x
 username: team-X
+custom_id: my_team_x # optional and not recommended, please use `username`
```

Comment:
If this is not recommended let's don't use it as an example?

Reply:
This is not just an example but also a source of specification.
We don't detail out our CRDs with these and hence users will read this page to figure out the `key`.

In the past, users have tried configuring custom_id using the key `customId` instead of `custom_id`.
We need to document it somewhere so I added it here.
The documents would need to be improved in future but right now, this is what we have.

Reply:
I was not understanding this comment correctly, makes sense to me!